metaflow/util.py

```diff
@@ -467,6 +503,19 @@ def to_pod(value):
     return str(value)
 
 
+def read_artifacts_module(file_path):
+    import importlib.util
+
+    try:
+        spec = importlib.util.spec_from_file_location("artifacts_module", file_path)
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
```

Comment:
Seems like if there's arbitrary python code in the file, it will be executed here. Do we need to worry about that?